install_files/ansible-base/roles/tails-config/tasks/migrate_existing_config.yml

```diff
@@ -0,0 +1,33 @@
+---
+# Between 0.3.12 and 0.4, the "Document Interface" was renamed to
+# "Journalist Interface". Since multiple scripts expect to readfrom this file,
+# we'll rename the old file to handle the transition gracefully.
+- name: Migrate Document Interface ATHS file (upgrade only).
+  command: >-
+    mv
+    "{{ tails_config_ansible_base }}/app-document-aths"
+    "{{ tails_config_ansible_base }}/app-journalist-aths"
+  args:
+    # Only run this command if the deprecated file is present.
+    removes: "{{ tails_config_ansible_base }}/app-document-aths"
+
+# Between 0.3.12 and 0.4, the `prod-specific.yml` vars file was deprecated
+# in favor of `group_vars/all/site-specific`. The latter is not version
+# controlled, and so will not exist by default.
+- name: Migrate site-specific configuration to new location.
+  command: >-
+    cp
```

Comment:
> Version controlling the template for the prod-specific.yml file has caused a lot of headaches around iterating on the configuration, since Admins must stash changes, pull the latest tag, then pop their changes, and pray there aren't any conflicts during pop.

Heh. Yeah thats why I want to kill that scene 🔪  :)  Good call on the `mv` breakage... ummm well, I think leaving the file in git is still confusing right? 

What about removing `prod-specific.yml` from tracking, add it to the `.gitignore` list, and `mv` it? Note that with your `creates` logic, the mv action will never happen if the file already exists so we shouldn't get a data loss situation. Does that make sense? If you want me to implement and test on tails let me know -- not trying to drag this PR down fixing side issues. Or ya I'm happy to punt this specific issue to a new issue/PR if it makes things easier.